Lib/test/libregrtest/win_utils.py

```diff
@@ -93,7 +103,7 @@ def getloadavg(self):
             # "07/19/2018 01:32:26.605","3.000000"
             toks = line.split(',')
             # Ignore blank lines and the initial header
-            if line.strip() == '' or (COUNTER_NAME in line) or len(toks) != 2:
+            if line.strip() == '' or '\\\\' in line or len(toks) != 2:
                 continue
 
             load = float(toks[1].replace('"', ''))
```

Comment:
While you're here, could you handle the case where `float` raises a ValueError (or maybe a TypeError? But I think we're guaranteed to only get ValueErrors at this point)? We can continue to the next line in that case.

Reply:
Hi zooba,
what about
```suggestion
            try:
                toks = line.strip().split(',')
                load = float(toks[-1].replace('"', ''))
            except Exception:
                continue
```
this would handle the ValueError/ TypeError as well as take care of the line parsing.

Reply:
@LorenzMende That looks like a good option to me. You'll have to merge the change yourself though - I don't have permissions to your PR branch.